proto/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Start with a NodeJS base image that also contains yarn.
-FROM node:12.17.0-buster as nodejsbuilder
+FROM node:16.14.2-buster as nodejsbuilder
```

Comment:
We also need to bump this version in the `Dockerfile` and `dev.Dockerfile`, otherwise I get `error react-scripts@5.0.0: The engine "node" is incompatible with this module. Expected version ">=14.0.0". Got "12.17.0"`

Reply:
Ah, good catch. I updated those two files in https://github.com/lightninglabs/lightning-terminal/pull/350/commits/0233f27f33262cf6aad4e792196dedb16e4ab33c